File: Nextpark/NextPark.Models/Models/UserModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace NextPark.Models
{
    public class UserModel : BaseModel
    {
        public double Coins { get; set; }
        public string Email { get; set; }
        public string UserName { get; set; }

        public string Name { get; set; }
        public string Lastname { get; set; }

        public string Address { get; set; }
        public string State { get; set; }
        public string CarPlate { get; set; }
        
    }
}

File: NextPark/NextPark.Web/src/app/models/user.model.ts
import { BaseModel } from './base.model';

export class User extends BaseModel {
  name: string;
  lastname: string;
  username: string;
  email: string;
  phone: string;
  address: string;
  cap: string;
  city: string;
  state: string;
  carPlate: string;
  balance: string;
  profit: string;
  token?: string;
}
